src/scripts/install.py

```diff
@@ -63,6 +63,55 @@ def log_level():
 
     return (options, args)
 
+def prepend_destdir(path):
+    """
+    Needed because os.path.join() discards the first path if the
+    second one is absolute, which is usually the case here. Still, we
+    want relative paths to work and leverage the os awareness of
+    os.path.join().
+    """
+    try:
+        destdir = os.environ['DESTDIR']
+    except KeyError as e:
+        destdir = ""
+
+    if destdir != "":
+        """
+        DESTDIR is non-empty, but we cannot join all prefix paths.
+
+        These will be rejected via an exception:
+          C:/foo
+          C:foo
+          \\foo (Python >3.1 only)
+          \\foo\bar (Python >3.1 only)
+          ../somewhere/else
+
+        These will be normalized to a relative path and joined with DESTDIR:
+          /absolute/dir
+          relative/dir
+          /dir/with/../inside
+          ./relative/to/me
+          ~/botan-install-test
+        """
```

Comment:
A Python docstring is always placed right below the symbol it documents, usually a method or class name. Here we have a stray string literal that is not assigned to anything. So in this case, you should use a usual comment format (all lines with #, there are no multi line comments in Python, sorry)